doc/source/building/index.rst

```diff
@@ -125,18 +125,18 @@ your system.
 
         brew install gfortran openblas pkg-config
 
-    .. note::
+    To allow the build tools to find OpenBLAS, you must run::
```

Comment:
I don't think it's a "must", the old text said "may need to" because it's supposed to work out of the box. It not doing so is a Homebrew packaging bug, and I'm not sure that that bug is present for all versions.

Reply:
I see - it doesn't work without that on my system, which is why I added that.

>It not doing so is a Homebrew packaging bug

Is this a Homebrew bug? My impression from reading `brew info openblas` is that it is intentionally not included in normal search paths:

>openblas is keg-only, which means it was not symlinked into /usr/local, because macOS provides BLAS in Accelerate.framework.

If this is an intentional choice on the part of the Homebrew authors, it seems likely to happen to all users going forward.

Reply:
Argh yes. I think it worked at one point, but this is a conscious decision, so okay to use "must" then.